src/Main.java

```diff
@@ -1,4 +1,82 @@
 public class Main {
     public static void main(String[] args) {
+        //Задание 1
+        System.out.println(" ");
```

Comment:
```suggestion
        System.out.println();
```
пробел можно не писать